packages/core/types/Fields.ts

```diff
@@ -56,7 +56,7 @@ export type ArrayField<
       : Field<Props[0][SubPropName], UserField>;
   };
   defaultItemProps?: Props[0];
-  getItemSummary?: (item: Props[0], index?: number) => string;
+  getItemSummary?: (item: Props[0], index?: number) => string | ReactNode;
```

Comment:
```suggestion
  getItemSummary?: (item: Props[0], index?: number) => ReactNode;
```

ReactNode includes strings, numbers, or anything that can be used as React children.